packages/artifact_proxy/bin/server.dart

```diff
@@ -0,0 +1,55 @@
+// ignore_for_file: avoid_print
+
+import 'dart:io';
+
+import 'package:args/args.dart';
+import 'package:artifact_proxy/artifact_proxy.dart';
+import 'package:shelf/shelf.dart';
+import 'package:shelf/shelf_io.dart' as shelf_io;
+import 'package:shelf_hotreload/shelf_hotreload.dart';
+import 'package:yaml/yaml.dart';
+
+Future<void> main(List<String> args) async {
+  final parser = ArgParser()
+    ..addOption(
+      'config',
+      defaultsTo: 'config.yaml',
+      help: 'Path to config file.',
+    )
+    ..addFlag(
```

Comment:
This appears to have been removed?

Reply:
Yup I removed the extra dependency on `package:args` and simplified things as much as possible.